src/server/hotspot_partition_calculator.cpp

```diff
@@ -136,7 +150,8 @@ void hotspot_partition_calculator::data_analyse()
             "The number of partitions in this table has changed, and hotspot analysis cannot be "
             "performed,in %s",
             _app_name.c_str());
-    for (int data_type = 0; data_type <= 1; data_type++) {
+
+    for (uint32_t data_type = 0; data_type <= 1; data_type++) {
```

Comment:
`1` is a magic number here, you can define an enum `HOTSPOT_DATA_TYPE_MAX` and use it here.